tools-public/toolpad/resources/functions.ts

```diff
@@ -107,6 +107,38 @@ SELECT * FROM final_table
   return data.data;
 }
 
+export async function getTeamPullRequestReviews(repo: string = 'mui/material-ui') {
+  const repoMap = {
+    'mui/material-ui': 23083156,
+    'mui/base-ui': 762289766,
+    'mui/pigment-css': 715829513,
+  };
+  const openQuery = `
+SELECT
+  DATE_FORMAT(created_at, '%Y-%m-01') AS t_month,
+  COUNT(*) AS cnt 
+FROM github_events ge
+WHERE
+  type = 'PullRequestReviewEvent' AND
+  action = 'created' AND
+  repo_id = ${repoMap[repo]} AND
+  actor_login IN ('mnajdova', 'siriwatknp', 'Janpot', 'DiegoAndai', 'mj12albert', 'aarongarciah', 'brijeshb42', 'michaldudak', 'colmtuite', 'atomiks', 'vladmoroz')
```

Comment:
I think to replace with https://github.com/mui/mui-public/blob/e5f56b32b99f2f67b9efa19ad2141dcd412673d4/tools-public/toolpad/resources/functions.ts#L269